migrations/versions/201506051421_4074211727ba_add_indico_unaccent_function.py

```diff
@@ -0,0 +1,37 @@
+"""ADD indico_unaccent function
+
+Revision ID: 4074211727ba
+Revises: 3f3a9554a6da
+Create Date: 2015-06-05 14:21:52.752777
+"""
+
+from indico.core.db.sqlalchemy.core import db
+
+# revision identifiers, used by Alembic.
+revision = '4074211727ba'
+down_revision = '3f3a9554a6da'
+
+
+def upgrade():
+    db.session.execute('''
+        CREATE OR REPLACE FUNCTION indico_unaccent(value TEXT)
+            RETURNS TEXT
+        AS $$
+        BEGIN
+           IF EXISTS(SELECT TRUE FROM pg_extension WHERE extname = 'unaccent')
+           THEN
+            RETURN unaccent(value);
+           ELSE
+                RETURN value;
+           END IF;
+        END;
+        $$
+        LANGUAGE plpgsql;
+    ''')
+    db.session.commit()
+
+def downgrade():
+    db.session.execute('''
+        DROP FUNCTION IF EXISTS indico_unaccent(TEXT);
+    ''')
+    db.session.commit()
```

Comment:
No trailing linebreak - please configure your editor to make it ensure one at the end of each file when saving it.